velox/docs/develop/aggregate-functions.rst

```diff
@@ -169,6 +184,14 @@ function's argument type(s) wrapped in a Row<> even if the function only takes
 one argument. This is needed for the SimpleAggregateAdapter to parse input
 types for arbitrary aggregation functions properly.
 
+Some function-level variables needs to be declared in the simple aggregation
+function class. These variables are initialized once when the aggregation
```

Comment:
nit: The author can optionally define function-level variables in the simple aggregation function class. These variables...